src/input_builders.jl

```diff
@@ -0,0 +1,105 @@
+"""
+    gustiness_constant(val)
+
+Create a constant gustiness parameterization with the given value.
+
+Returns a `ConstantGustinessSpec(val)` that provides a fixed gustiness 
+contribution to the effective windspeed.
+"""
+gustiness_constant(val) = ConstantGustinessSpec(val)
```

Comment:
Do we need this constructor? (In the tests you use `gustiness_constant` in only two cases, and use `ConstantGustinessSpec` in all other cases.)

Reply:
Removed